google-cloud-storage/src/test/resources/META-INF/native-image/com/google/cloud/storage/proxy-config.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "interfaces":["com.google.cloud.storage.spi.v1.StorageRpc"]
```

Comment:
Would you add comment explaining why this configuration is not in main?

Reply:
Thanks for point this out! I realized that this class also needs to be in the main config directory since it is referenced outside of tests. For example:
https://github.com/googleapis/java-storage/blob/3716105b5ea83378e5cacb4c4cd480b6dfe0fece/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java#L90
Apologies for the confusion @BenWhitehead!